contract/models/mail_message.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class MailMessage(models.Model):
+    _inherit = 'mail.message'
+
+    def message_format(self):
+        result = super().message_format()
+        contract_modification = self.env['ir.model.data'].xmlid_to_res_id(
+            'contract.mail_message_subtype_contract_modification'
+        )
+        for message in result:
+            message.update({
```

Comment:
Does this makes that the outgoing mail is seen as external?

Reply:
Appear as internal yet.
I have only applied commit related to https://github.com/OCA/contract/pull/613#issuecomment-766145055 but probably isn't final solution.

Reply:
Can you avoid to rewrite in the message and do this change at other level?